hw/xbox/xbox.c

```diff
@@ -476,6 +476,28 @@ static void machine_set_eeprom(Object *obj, const char *value,
     ms->eeprom = g_strdup(value);
 }
 
+static char *machine_get_avpack(Object *obj, Error **errp)
+{
+    XboxMachineState *ms = XBOX_MACHINE(obj);
+
+    return g_strdup(ms->avpack);
+}
+
+static void machine_set_avpack(Object *obj, const char *value,
+                               Error **errp)
+{
+    XboxMachineState *ms = XBOX_MACHINE(obj);
+
+    if (!smbus_avpack_to_reg(value, NULL))
```

Comment:
This is a statically defined function in smbus_xbox_smc.c. We should make it public if we are calling it here

Reply:
Indeed, thanks.